### ISSUE TITLE
View Vignetting / Tunneling / Simulator Sickness Prevention

### ISSUE THREAD

**Issue by Pinkyyyy** [Yellow-Dog-Man/Resonite-Issues#404]
### Is your feature request related to a problem? Please describe.

I love using the teleport locomotion, it's the one that gives me the least amount of motion sickness but sometimes it can be frustrating when you only want to move a tiny bit. Switching to a walking locomotion works, however even just using it for tiny movements makes me feel awful, a big part is just that I am very new to VR but another is that there is no comfort options at all, no vignette when moving, or even just an option to make walking snapping like how turning works.

### Describe the solution you'd like

- The ability to disable motion blur (#178) / other graphical effects
- Vignette filter when moving
- Option for discrete physical locomotion (shift/bump/step/increment) in place of continuous locomotion (similar to snap vs smooth turning)

### Describe alternatives you've considered

I already use an alternative method to change settings, such as installing the Resonite Mod Loader and Photonic Freedom mod and you can use user created content to achieve the Vignette effect or just program it yourself in the game.

**Comment by Psychpsyo** [comment]
Motion blur is already covered by #178

**Comment by Azavit** [comment]
**Vignette filter when moving**
Is a big one I'd say, I keep on meaning to make my own version of this in platform but keep forgetting to/other projects that grab my attention more.

Having a filter like this is how I actually got my VR legs to begin with (Township Tale was the game) and I do believe it should be a standard _option_ for all VR titles that expect people who are new to VR.

**Comment by FlameSoulis** [comment]
Bumping this as I had reported this previously on Neos's Github, so linking to complete any relevant issues from there are mentioned here: https://github.com/Neos-Metaverse/NeosPublic/issues/3427

**Comment by TisFoolish** [comment]
It says in the best practices part of the issue tracker's Readme to not refer to projects the devs worked on in the past they're unable to use materials or issues from past projects.

**Comment by FlameSoulis** [comment]
> It says in the best practices part of the issue tracker's Readme to not refer to projects the devs worked on in the past they're unable to use materials or issues from past projects.

Just re-read it. Nuked the comment and crossing off that task from the checklist.

**Comment by Stellanora64** [comment]
@Geenz is this no longer blocked?
